VerovioFramework/VerovioFramework.h

```diff
@@ -0,0 +1,91 @@
+//
+//  VerovioFramework.h
+//  VerovioFramework
+//
+//  Created by Thomas Ramson on 06.06.19.
+//
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for VerovioFramework.
+FOUNDATION_EXPORT double VerovioFrameworkVersionNumber;
+
+//! Project version string for VerovioFramework.
+FOUNDATION_EXPORT const unsigned char VerovioFrameworkVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <VerovioFramework/PublicHeader.h>
```

Comment:
Can't you use wildcards for this? It is annoying to have to add each new header by hand. In other builds (Python, Java, etc.*) we do not have to do it.

Reply:
I'm sorry but Objective-C doesn't offer any kind of wildcard imports.
It could be possible to ignore the imports within the umbrella file. If you import a file that is not added to the umbrella header will lead to a warning `Missing submodule '...'`.

You also have to include every new file within the new framework target. 

If you don't want to do this I could do the maintenance part with adding new files to the framework target + adjusting the umbrella header file.

Reply:
It could be possible to add a script to add all new files to the framework target + adjust the umbrella header. But in this case all files, needed or not, will be added to the project as well.

Reply:
> It could be possible to add a script to add all new files to the framework target + adjust the umbrella header

This sounds good to me.

Please move the output VerovioFramework to a `./bindings/ios` directory?

Reply:
It looks like the script approach is not possible. There is no documented way to add files via script or command-line tool to the project file. The project file can become corrupted with any new Xcode version.
I also googled for another approach but could not find one. It's always needed to manually add the file to the target and adjust the visibility level (private, project, public). Adding the import to the umbrella header just fixes the warning.

If you can't approve this pull request I will continue with the fork and adapt all changes there.

Reply:
I moved the VerovioFramework files (VerovioFramework.h + Info.plist) to `./bindings/iOS`directory